tests/saitests/py3/switch.py

```diff
@@ -906,6 +906,15 @@ def sai_thrift_read_buffer_pool_watermark(client, buffer_pool_id):
     return wm_vals[0]
 
 
+def sai_thrift_clear_buffer_pool_watermark(client, buffer_pool_id):
```

Comment:
where is this func used?

Reply:
Planned to use in class BufferPoolWatermarkTest.

The new RPC API is merged in https://github.com/opencomputeproject/SAI/pull/1804, I still need to figure out how SONiC get this update before moving forward.